test/WebJobs.Script.Tests.Integration/WebJobs.Script.Tests.Integration.csproj

```diff
@@ -42,9 +42,7 @@
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="8.0.1" />
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.11.2" />
     <PackageReference Include="Microsoft.Azure.EventHubs" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Azure.Functions.NodeJsWorker" Version="3.10.0" />
```

Comment:
The tests are not using anything from these packages?

Reply:
They are now brought in through `worker.props` at the top.